### ISSUE TITLE
right TDB

### ISSUE THREAD

**Issue by f3rret** [LegionEmulationProject/TrinityCore#59]
Thank you for this great project. I hope it will be better every day.
But some questions are unclear. After propper install & configure I still have many errors on dberror.log such as:

_GameObject (Entry: 215017 GoType: 8) have data0=5756392 but SpellFocus (Id: 5756392) not exist.
Quest 39311 objective 281459 has non existing gameobject entry 243364, quest can't be done.
Quest 31017 has `ItemDrop1` = 82809 but item with entry 82809 does not exist, quest can't be done.
Spell Id 208764 has DBC ProcFlags 16384, but it's of non-proc aura type, it probably needs an entry in `spell_proc` table to be handled correctly.
Table `creature` has creature (GUID: 73852 Entry: 21033 MapID: 530) spawned on a possible invalid position (X: 2180.790039 Y: 6149.399902 Z: 145.955002)
table `lfg_entrances` contains coordinates for wrong dungeon 18
table `lfg_entrances` contains coordinates for wrong dungeon 165
Failed to load dungeon Бесконечные залы (Id: 1583), cant find areatrigger for map 1764_

and much more. I use TDB_world_735.00_2018_02_19.sql  and TDB_hotfixes_735.00_2018_02_19.sql from master branch releases of TrinityCore project. Is this right way or its better to use TDB from another place?

**Comment by TheGhostGroup** [comment]
@f3rret  You are using the correct database. There is a lot of DB errors due to a lack of work by trinitycore on the database. I have been working to reduce the errors between adding new spawns for new content. I'm the only one working on this project so fixes are added when i can. It take a lot of time to craft patch fixes.

**Comment by f3rret** [comment]
One more stupid question. As I understand there is no mobs yet in Pandaria, Broken Islands and Draenor locs. Dungeons also not working at this locations. Is this correct or I have some trouble with my configuration?

**Comment by TheGhostGroup** [comment]
@f3rret You are correct there are no mobs in Pandaria, Broken Islands or Draenor, Also no dungeons for those expansions are working yet. But I'm working on spawning the dungeons and scripting them but it will take a LOT of time before they will be usable.

**Comment by f3rret** [comment]
Thank you for answers and for this great project. We look forwards for new updates!
